daemon/kill.go

```diff
@@ -105,6 +115,9 @@ func (daemon *Daemon) Kill(container *container.Container) error {
 		return errNotRunning{container.ID}
 	}
 
+	// FIXME(vdemeester) should we disable restartpolicy for SIGKILL or not ?
```

Comment:
From my POV I would expect the RestartPolicy to only be disabled if the kill was generated as part of a `docker stop` command.

WDYT?